docs/changelog/4.2.rst

```diff
@@ -0,0 +1,1146 @@
+Changelogs for 4.2.x
+====================
+
+.. changelog::
+  :version: 4.2.0
+  :released: *unreleased*
+
+  This release *blah blah blah* …
+
+  Major features for this release:
+
+  * Lua records
+  * IXFRDist
+  * API changes
+
+  Minor features, weave paragraph from this:
+
+  * 6823: On incoming NOTIFY load our serial from backend to have it
+  * 6969: re-do of an earlier PR to add the glorious log-log histograms
+  * 6997: Name threads in the programs
+  * 6727: ALIAS: Respond SERVFAIL on non-NOERRORs from resolver
+  * 6910: support for OpenSSL 1.1.1's ed25519 and ed448
+  * 6975: Enhance query-logging with timing
+  * 6913: actually truncate truncated responses
```

Comment:
bugfix

Reply:
Resolving under the assumption you were only talking about #6913.

Reply:
yes